code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -99,6 +99,7 @@
 		M.current.weakened = 0
 		M.current.stunned = 0
 		M.current.paralysis = 0
+		M.current.reagents.clear_reagents()
```

Comment:
I could do that, or I could increase the cooldown on the skill, most vampires I see killed end up burning in the chapel.